util/kvcache/simple_lru.go

```diff
@@ -118,6 +118,9 @@ func (l *SimpleLRUCache) Put(key Key, value Value) {
 		if l.size > l.capacity {
 			lru := l.cache.Back()
 			l.cache.Remove(lru)
+			if l.onEvict != nil {
```

Comment:
Any unit test for this?

Reply:
Having a single test, not pushed yet.

Reply:
Actually yes, it's in util/kvcache/simple_lru_test.go TestPut() function.
There was a PR already merged last week:
PR #24242

Reply:
No, the unit test added-in #24242 doesn't cover this, since the code is a new addition in this PR.

Reply:
This is a bug fix.
If not added, kvcache.onEvict() will not be called when quota==0.

Reply:
I mean, add a unit test for this bug fix.

Reply:
@crazycs520 test added in util/kvcache, PTAL.